### ISSUE TITLE
External Asset Pipeline strips sourcemap from stylesheets when merging into sitemap

### ISSUE THREAD

**Issue by egermani** [middleman/middleman#2135]
I'm trying to enable css sourcemaps in my Middleman project. As with [#2096](https://github.com/middleman/middleman/issues/2096), adding `config[:sass_source_maps] = true` does nothing. So I went to Webpack and its ExtractTextPlugin to generate my stylesheet. This worked! My external asset pipeline goes to `.tmp/dist`, and when I run `middleman build`, that folder now has a stylesheet that has a sourcemap at the bottom. (Which looks like this: `/*# sourceMappingURL=webpack-site.css.map*/`). But when I look at the `build` folder, that file is not identical to its source in the `.tmp/dist` folder. It has all the CSS rules... and not the sourcemap.

What does middleman do while merging the external asset source into the sitemap that would account for this truncation of my sourcemapped file? My first thought was that it simply removes any comments from your stylesheets on build, but a multi-line comment in the Sass was able to make it into the build folder unscathed.

So, to recap: Middleman seems to be yanking out the sourcemap declaration at the bottom of my compiled CSS file for some reason.

I should note that the Javascripts do not get their source maps removed.

Any advice would be great. This is the kind of thing that would make me move on to a different tool, and I'd prefer to continue using Middleman for static site development.

## Additional information

- Ruby version: 2.4.2p198
- Middleman version: 4.2.1
- OS version: macOs High Sierra (Version 10.13.1)
